packages/shared/src/styles/themes.scss

```diff
@@ -142,6 +142,8 @@
         --status-transfer: #{$color-orange};
         --status-info: #{$color-blue};
         --status-colored-background: #{$color-white};
+        --status-in-active: #{$color-red-6};
```

Comment:
`inactive` is one word, so this should be `--status-inactive`.

Reply:
Will make this change